ethportal-peertest/src/lib.rs

```diff
@@ -54,7 +54,7 @@ async fn launch_node(trin_config: TrinConfig) -> anyhow::Result<PeertestNode> {
 }
 
 fn generate_trin_config(id: u16, bootnode_enr: Option<&Enr>) -> TrinConfig {
-    let discovery_port: u16 = 9000 + id;
+    let discovery_port: u16 = 9009 + id;
```

Comment:
I would replace this value (`9009`) with an import of `DEFAULT_DISCOVERY_PORT`, it'll make life easier down the road if we ever need to change the default again. And similarly in `main.rs` / `overlay_service.rs`

Reply:
done